conda/plugins/types.py

```diff
@@ -184,3 +185,22 @@ class CondaPostSolve:
 
     name: str
     action: Callable[[str, tuple[PackageRecord, ...], tuple[PackageRecord, ...]], None]
+
+
+@dataclass
+class CondaConfigurationParameter:
+    """
+    Return type to use when defining a conda configuration parameter plugin hook.
+
+    For details on how this is used, see
+    :meth:`~conda.plugins.hookspec.CondaSpecs.conda_post_solves`.
```

Comment:
I think this reference points to a different thing.

Reply:
ahh, left over from copy and paste. Thanks for catching it.